doc/user/tutorials/video-series-overview.md

```diff
@@ -58,23 +58,12 @@ If you are not familiar with geographic information systems (GIS) and geospatial
 - Calculate the surface volume area of a raster layer
 
 ```{note}
-In our example, ‘select by attribute’ for a population greater than 5000 would only work
-on an integer field. In v2.0 and later, the population size field is a text string and
-you will first need to use the field calculator to convert the population field into an
-integer. To do so:
+The "Populated places" layer referred to in this video has been updated and moved from "Places/Community map
+(crowd-sourced)" to the root of the "Places/" layer group in QGreenland v3.
 
-1. Open up the attribute table, and click the ‘Open field calculator’ icon.
-
-    ![OpenFieldcalculator_Larger](/_images/OpenFieldcalculator_Larger.png)
-
-2. Check ‘Create a new field’. Designate an output field name (example: pop_numeric) and
-   select the output field type ‘Integer (32 bit)’. In the expression panel, choose the
-   field ‘population’. Click OK.
-
-    ![create_new_field](/_images/create_new_field.png)
+This updated "Populated places" layer has a "Population 2016" attribute that should be used instead of the "population" attribute indicated in the video.
```

Comment:
Is this updated in the YouTube video description too?

Reply:
Just updated!